core/src/com/unciv/logic/civilization/EspionageManager.kt

```diff
@@ -0,0 +1,45 @@
+package com.unciv.logic.civilization
+
+import com.unciv.logic.IsPartOfGameInfoSerialization
+
+class Spy(val name: String) : IsPartOfGameInfoSerialization {
+    fun clone(): Spy {
+        return Spy(name)
+    }
+}
+
+class EspionageManager : IsPartOfGameInfoSerialization {
+
+    var spyCount = 0
+    var spyList = mutableListOf<Spy>()
+    var erasSpyEarnedFor = mutableListOf<String>()
```

Comment:
I feel like this should be a `by lazy` method that looks at various uniques (e.g., English get 2 spies initially compared with the 1 every other civ gets). This change could come later though once more of the espionage system is in place.